src/data/phase/dto.type.ts

```diff
@@ -3,7 +3,7 @@ export type Duration = Readonly<{
   to: string
 }>
 
-export type TaskContentId = 'rewards' | 'criteria' | 'submit' | 'description'
+export type TaskContentId = 'rewards' | 'criteria' | 'submit' | 'description' | 'documentation'
```

Comment:
If `documentation` is not used by the tasks, please remove it from the type union and create another type.